xonsh/environ.py

```diff
@@ -81,7 +110,8 @@ def _replace_home(x):
 FORMATTER_DICT = dict(user=os.environ.get('USER', '<user>'),
                       hostname=socket.gethostname().split('.', 1)[0],
                       cwd=lambda: _replace_home(builtins.__xonsh_env__['PWD']),
-                      curr_branch=lambda: current_branch(),
```

Comment:
I would still like to keep a function called `current_branch()` so it is obvious where to add the functionality for hg, svn, etc.